docs/getting-started-guide.md

```diff
@@ -0,0 +1,168 @@
+# How to set up and run the Co-Simulation Demo
+
+This demonstration shows how to compile and run the Co-Simulation demo of Buildroot in QEMU with a simulated device in SystemC. This configeration is tested working for Ubuntu 18.0.4 and assumes that `cosim` is located in your home directory. This walkthough also assumes the device being emulated is the Zynq-7000 SoC.
```

Comment:
Using Buildroot to make the OS environment is just one way to set up that part.  For this example, it might be worth noting that Buildroot was selected so that a pure upstream Linux kernel can be built and deployed in QEMU.  This keeps the setup simple and shows how there isn't anything Petalinux specific in creating a complete solution.

Reply:
That is a good distinction. All of this should also be compatible with Petalinux or any other env in theory.